context_test.go

```diff
@@ -303,3 +306,44 @@ func TestLoadProjectNoSrcDir(t *testing.T) {
 		t.Fatalf("should have returned 'Split Absolute Root' error (no 'src' dir present)")
 	}
 }
+
+// TestCaseInsentitive is test for Windows. This should work even though set
+// difference letter cases in GOPATH.
+func TestCaseInsentitiveGOPATH(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.Skip("skip this test on non-Windows")
+	}
+
+	h := test.NewHelper(t)
+	defer h.Cleanup()
+
+	h.TempDir("src")
+	h.TempDir("src/test1")
+	h.TempFile("src/test1/manifest.json", ` "dependencies":{} `)
+	h.TempFile("src/test1/lock.json", `{"memo":"cdafe8641b28cd16fe025df278b0a49b9416859345d8b6ba0ace0272b74925ee","projects":[]}`)
```

Comment:
nit: I think this should be able to be dropped from this test - only the manifest is required for root detection to work.